firstvoices/backend/views/import_job_views.py

```diff
@@ -159,3 +193,49 @@ def confirm(self, request, site_slug=None, pk=None):
         return Response(
             serializer.data, status=status.HTTP_202_ACCEPTED, headers=headers
         )
+
+    @action(detail=True, methods=["post"])
+    def validate(self, request, site_slug=None, pk=None):
+        """
+        Method to start the validation process on a given import-job.
+        """
+        site = self.get_validated_site()
+        import_job_id = self.kwargs["pk"]
+        logger = logging.getLogger(__name__)
+
+        # Verify that no other jobs are started or queued for the same site
+        existing_incomplete_jobs = get_import_jobs_queued_or_running(
+            site, import_job_id
+        )
+
+        if len(existing_incomplete_jobs):
+            logger.error(
```

Comment:
If this message is meant for the user, it needs to go in the Response object instead of into the logs.